engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/ReachEndOfLogTest.java

```diff
@@ -56,7 +56,7 @@ public void shouldReturnFalseIfNotReachedEndOfLog() {
         .withXmlResource(
             Bpmn.createExecutableProcess("process")
                 .startEvent()
-                .manualTask("test")
+                .intermediateThrowEvent("test")
```

Comment:
:wrench: We could also disable the flag

Reply:
I thought about that, but there is no way to switch a single flag currently. So that means a compilation issue in this test every time we add a feature flag. We already have it in the other test, wanted to prevent it in other places.